File: SpringBootCustomJackson/src/main/java/com/example/java/SpringBootCustomJacksonApplication.java
package com.example.java;

import java.util.Arrays;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.ApplicationContext;

@SpringBootApplication
public class SpringBootCustomJacksonApplication {

	public static void main(String[] args) {
		ApplicationContext ctx = SpringApplication.run(SpringBootCustomJacksonApplication.class, args);
		
		System.out.println("Let's inspect the beans provided by Spring Boot:");

		Arrays.stream(ctx.getBeanDefinitionNames()).sorted().forEach(System.out::println);
	}
}

File: SpringBootCustomJackson/src/main/kotlin/com/example/kotlin/SpringBootCustomJacksonApplication.kt
package com.example.kotlin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import java.util.*

@SpringBootApplication
class SpringBootCustomJacksonApplication

fun main(args: Array<String>) {

    val context = runApplication<SpringBootCustomJacksonApplication>(*args)

    println("Let's inspect the beans provided by Spring Boot:")

    context.beanDefinitionNames.asList().sorted().forEach { println(it) }
}
